docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -31,7 +31,7 @@ To add additional clusters, increment the `0` value for each variable (`CAMUNDA_
 | `CAMUNDA_MODELER_CLUSTERS_0_ID`             | A unique identifier to use for your cluster.                                                                   | `test-cluster-1`                                                            |
 | `CAMUNDA_MODELER_CLUSTERS_0_NAME`           | The name of your cluster.                                                                                      | `Test Cluster 1`                                                            |
 | `CAMUNDA_MODELER_CLUSTERS_0_VERSION`        | The Camunda version used by this cluster.                                                                      | `8.7.0`                                                                     |
-| `CAMUNDA_MODELER_CLUSTERS_0_AUTHENTICATION` | The authentication to use with your cluster.                                                                   | `OAUTH`, `NONE`                                                             |
+| `CAMUNDA_MODELER_CLUSTERS_0_AUTHENTICATION` | The authentication to use with your cluster.                                                                   | `BEARER_TOKEN`, `NONE`                                                      |
```

Comment:
Should we provide a link to the announcement below this table? I guess it depends how much it may confuse users.

Reply:
I'd say for the 8.8 version of the docs we don't necessarily need a link to the announcement, but for 8.7 it makes sense.